.github/workflows/check-dist.yml

```diff
@@ -0,0 +1,52 @@
+# `dist/index.js` is a special file in Actions.
+# When you reference an action with `uses:` in a workflow,
+# `index.js` is the code that will run.
+# For our project, we generate this file through a build process from other source files.
+# We need to make sure the checked-in `index.js` actually matches what we expect it to be.
+name: Check dist/
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This repo doesn't use a `main` branch (yet)! 
Let's fix that outside this PR.

Reply:
Fixed!